### PR TITLE
Rename jsonFetcher to jsonStatham

### PR DESCRIPTION
**Pull request by maneeshanand**
@everplans/fluxxed_up 

please review!

This PR changes the name of `jsonFetcher` to `jsonStatham`.  `Transporter` jokes are welcome.  Please refrain from joking about `Snatch` since that movie is awesome.

### CODE REVIEW

**Comment by bartriordan** [comment]
Testing the new Slackbot response I set up @everplans/fluxxed_up

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export ActionPrototype from './lib/ActionPrototype'
 export ExtraStorage from './lib/extra_storage'
 export Dispatcher from './lib/fu-dispatcher'
-export JsonFetcher from './lib/jsonFetcher'
+export JsonStatham from './lib/jsonStatham'
```
Comment (bartriordan) [inline src/index.js]: @maneeshnycda Inconsistent capitalization. It should always be camelCase not PascalCase since we don't call `new` before it.

**Comment by bartriordan** [comment]
@maneeshnycda One minor change necessary.

**Comment by whabib** [comment]
👍

**Pull request merged**